plugins/azure/virtualnetworks/networkGatewaysUsage.js

```diff
@@ -0,0 +1,92 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'Network Gateways Usage',
+    category: 'Virtual Networks',
+    description: 'Ensures that Virtual Networks are using subnets and network security groups instead of virtual network gateways.',
+    more_info: 'To meet your organization\'s security compliance requirements.',
```

Comment:
```suggestion
    more_info: 'Use subnets and network security groups to control network traffic instead of using virtual network gateways to meet your organization\'s security and compliance requirements.',
```